cmd/query/app/query_parser.go

```diff
@@ -134,6 +135,12 @@ func (p *queryParser) parseTraceQueryParams(r *http.Request) (*traceQueryParamet
 		return nil, err
 	}
 
+	if val, ok := tags[otelsemconv.OtelStatusCode]; ok {
```

Comment:
is this a merge error? Seems unrelated to your PR. When merging main either do it via GitHub button or via running `git rebase main` from your branch (preferred).